src/Queues/CLI/Process.php

```diff
@@ -63,6 +63,8 @@ public function run_command( $args, $assoc_args ) {
 
 		$endtime = time() + $this->timelimit;
 
+		\WP_CLI::log( sprintf( 'Processing queue %s. This will run for %d seconds. Run with --debug to see additional details.', $queue_name, $this->timelimit ) );
```

Comment:
If `$this->timelimit` is set to `0`, this will run indefinitely. Should this have a different message in that case?

Can we wrap the queue name in quotes or put it after a colon?

Reply:
Update the logic to account for infinite runs.